sdk/core/azure-core/azure/core/polling/base_polling.py

```diff
@@ -377,6 +428,7 @@ def __init__(
         lro_algorithms=None,
         lro_options=None,
         path_format_arguments=None,
+        operation_state=None,
```

Comment:
You canjust say `operation_state=DefaultOperationState()` and you wouldn't needt he additional settings in line 450, it'll just be `self._operation_state = operation_state`

Reply:
Done!